components/dashboard/src/components/Alert.tsx

```diff
@@ -123,14 +122,14 @@ export default function Alert(props: AlertProps) {
             {props.closable && (
                 <span className={`mt-1 ml-4`}>
                     {/* Use an IconButton component once we make it */}
-                    <Button
-                        type="secondary"
-                        className="bg-transparent hover:bg-transparent"
+                    <button
+                        type="button"
+                        className="bg-transparent p-1"
                         onClick={handleClose}
                         autoFocus={autoFocusClose}
                     >
                         <XSvg className="w-3 h-4 cursor-pointer" />
-                    </Button>
+                    </button>
```

Comment:
**thought**: We might want to keep these changes around, but should be fine to revert in this PR as we only use the closable prop in two instances, see [relevant diff](https://github.com/search?q=repo%3Agitpod-io%2Fgitpod+closable%3D%7Btrue%7D+&type=code). 🏓

Reply:
Let's keep them right away!